.circleci/config.yml

```diff
@@ -26,6 +23,8 @@ jobs:
       - run:
           name: Build and run unit tests
           command: mvn $MAVEN_CLI_OPTS package
+      - store_artifacts:
+          path: target/
```

Comment:
Why did you upload the entire target directory? and not only the jar files for example.